spec/example-artifacts.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "system": "OFM",
+    "name": "com.oracle.ofm",
+    "shortName": "Oracle Fusion Middleware",
+    "description": "Oracle's complete family of application infrastructure products—from the #1 Java application server to SOA and enterprise portals—are integrated with Oracle Applications and technologies to speed implementation and lower the cost of management and change. Best-of-breed offerings and unique hot-pluggable capabilities provide a foundation for innovation and extend the business value of existing investments.",
+    "url": "",
+    "vendor": "Oracle",
+    "vendorContact": "support@oracle.com",
+    "version": "11.1.1",
+    "deploymentSpec":{
```

Comment:
deploymentSpec and runtimeSpec appear redundant.  In my opinion we could simply have a host reference, and a provides reference (which would be a vector, perhaps of export objects).